web/src/javascript/gameLeaderboard.js

```diff
@@ -23,3 +27,31 @@ export function drawLeaderboard(state) {
   const old = document.getElementById("leaderboard-body");
   leaderboard.replaceChild(new_tbody, old);
 }
+
+function focusOnPlayer(id, state){
```

Comment:
How about (or similar):
```js
const cellToFocus = findBiggestCell(state.players.find(player => player.id == id).cells);
```
? Or using `.find` in `findPlayerWithId` if you prefer.